Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -125,6 +144,22 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
             self.nonceCompletionBlock(nonce)
         }
     }
+    
+    // MARK: - Helpers
+    
+    private func buttonsStackView(label: String, views: [UIView]) -> UIStackView {
+        let titleLabel = UILabel()
+        titleLabel.text = label
+        
+        let buttonsStackView = UIStackView(arrangedSubviews: [titleLabel] + views)
+        buttonsStackView.axis = .vertical
+        buttonsStackView.distribution = .fillProportionally
+        buttonsStackView.backgroundColor = UIColor.systemGray6
+        buttonsStackView.layoutMargins = .init(top: 5, left: 5, bottom: 5, right: 5)
```

Comment:
Take it or leave it, but to me at least it generally makes it clearer to use `UIEdgeInsets` instead of `.init`, it's the same thing so if you feel strongly feel free to leave as is.
```suggestion
        buttonsStackView.layoutMargins = UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5)
```